### PR TITLE
Add blog post about Log4j Zero Day

### PR DESCRIPTION
**Pull request by knaufk**

### CODE REVIEW

**Review by MartijnVisser — APPROVED** [review]
Looks good to me

File: _posts/2021-12-10-log4j-cve.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+- knaufk:
+  name: "Konstantin Knauf"
+excerpt: "Advise on Apache Log4j Zero Day (CVE-2021-44228)"
+---
+
+Yesterday, a new Zero Day for Apache Log4j was [reported](https://www.cyberkendra.com/2021/12/apache-log4j-vulnerability-details-and.html). 
+It is by now tracked under [CVE-2021-44228](https://nvd.nist.gov/vuln/detail/CVE-2021-44228). 
+
+Apache Flink is bundling a version of Log4j that is affeced by this vulnerability. 
+We recommend users to follow the [adivsory](https://logging.apache.org/log4j/2.x/security.html) of the Apache Log4j Community.
```
Comment (zentol) [inline _posts/2021-12-10-log4j-cve.md:15]:
```suggestion
We recommend users to follow the [adivsory](https://logging.apache.org/log4j/2.x/security.html) of the Apache Log4j Community. 
```

File: _posts/2021-12-10-log4j-cve.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  name: "Konstantin Knauf"
+excerpt: "Advise on Apache Log4j Zero Day (CVE-2021-44228)"
+---
+
+Yesterday, a new Zero Day for Apache Log4j was [reported](https://www.cyberkendra.com/2021/12/apache-log4j-vulnerability-details-and.html). 
+It is by now tracked under [CVE-2021-44228](https://nvd.nist.gov/vuln/detail/CVE-2021-44228). 
+
+Apache Flink is bundling a version of Log4j that is affeced by this vulnerability. 
+We recommend users to follow the [adivsory](https://logging.apache.org/log4j/2.x/security.html) of the Apache Log4j Community. 
+For Apache Flink this currently translates to "setting system property "log4j2.formatMsgNoLookups" to 'true'" until Log4j has been upgraded to 2.15.0 in Apache Flink.
```
Comment (zentol) [inline _posts/2021-12-10-log4j-cve.md]:
```suggestion
For Apache Flink this currently translates to "setting system property `log4j2.formatMsgNoLookups` to `true`" until Log4j has been upgraded to 2.15.0 in Apache Flink. 
```

**Review by zentol — APPROVED** [review]

File: _posts/2021-12-10-log4j-cve.md (hunk 1 above)
Comment (knaufk) [reply _posts/2021-12-10-log4j-cve.md:15]: ?

Comment (zentol) [reply _posts/2021-12-10-log4j-cve.md:15]: adivsory -> advisory

**Pull request merged**